backend/entityservice/serialization.py

```diff
@@ -41,11 +41,12 @@ def binary_format(encoding_size):
 
     The binary format string can be understood as:
     - "!" Use network byte order (big-endian).
+    - "I" store the entity ID as an unsigned int
```

Comment:
I guess we aren't going to be handling 4 billion entities in a single run anytime soon.

Reply:
our goal is 100 million. If there happens to be demand for more, we can fairly easily make adjustments.

Reply:
yeah I agree